packages/account/src/Components/poi-idv-document-submit/__tests__/poi-idv-document-submit.spec.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { isDesktop, isMobile } from '@deriv/shared';
+import IdvDocumentSubmit from '../poi-idv-document-submit';
+
+jest.mock('../../../Assets/ic-document-submit-icon.svg', () => jest.fn(() => 'DocumentSubmitLogo'));
+jest.mock('../utils.js', () => ({
+    getDocumentData: function (country_code, key) {
+        const data = {
+            tc: {
+                document_1: {
+                    new_display_name: '',
+                    example_format: '12345-A',
+                    sample_image: '',
+                },
+                document_2: {
+                    new_display_name: '',
+                    example_format: 'A-54321',
+                    sample_image: '',
+                },
+            },
+        };
+        return data[country_code][key];
+    },
+    getRegex: jest.fn(() => /a-54321/i),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isDesktop: jest.fn(() => true),
+    isMobile: jest.fn(() => false),
+    formatInput: jest.fn(() => 'A-54321'),
```

Comment:
The component IdvDocumentSubmit doesn't have isMobile, isDesktop imports. Do we need them?

Reply:
I did it just for covering with test native select input instead of our autocomplete. Because I could test internal formik onItemSelection function existing in autocomplete input for desktop version.